p/themes/Mapco/mapco.scss

```diff
@@ -51,3 +51,7 @@ a, button.as-link {
 	outline: none;
 	color: variables.$main-first;
 }
+
+.icon {
+	height: inherit;
+}
```

Comment:
What is the reason for this line?

Reply:
It overwrites a rule in `p/themes/base-theme/frss.css` that vertically scales the image icon, and distort it.

Reply:
https://github.com/FreshRSS/FreshRSS/blob/edge/p/themes/base-theme/frss.css#L726

Reply:
ah I see. I would prefer to improve frss.css and delete the `width` line and change `height: 1rem` to `height: inherit`. `height` and `width` were `16px` before I changed it to `1rem`. It should not be the issue of the themes to handle it by default.